app/models/partners/profile.rb

```diff
@@ -108,5 +115,26 @@ class Profile < Base
     def has_no_social_media?
       website.blank? && twitter.blank? && facebook.blank? && instagram.blank?
     end
+
+    def client_share_total
+      tot = 0
+      served_areas.each do |served_area|
+        tot += served_area.client_share
+      end
+      tot
```

Comment:
This can be a 1-liner :)

```suggestion
served_areas.sum(&:client_share)
```